src/components/FlowRunAutomation.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div v-if="automationId" class="flow-run-automation">
+    <span>Automation</span> <AutomationIconText :automation-id />
+  </div>
+</template>
+
+<script setup lang="ts">
+  import { addMinutes, subMinutes } from 'date-fns'
+  import { computed } from 'vue'
+  import AutomationIconText from '@/components/AutomationIconText.vue'
+  import { useWorkspaceEvents } from '@/compositions/useWorkspaceEvents'
+  import { FlowRun } from '@/models/FlowRun'
+  import { getResourceIdParts } from '@/utilities/events'
+
+  const props = defineProps<{
+    flowRun: FlowRun,
+  }>()
+
+  const { events } = useWorkspaceEvents(() => {
+    if (!props.flowRun.expectedStartTime) {
+      return null
+    }
+
+    // purposefully adding some margin around the expectedStartTime
+    // because the automation being executed should happen prior to the time the run was scheduled
+    const until = addMinutes(props.flowRun.expectedStartTime, 5)
+    const since = subMinutes(until, 5)
```

Comment:
```suggestion
    const since = subMinutes(props.flowRun.expectedStartTime, 5)
```
?

or isn't this otherwise the same as:
```suggestion
    const since = props.flowRun.expectedStartTime
```
?

Reply:
I updated this in https://github.com/PrefectHQ/prefect-ui-library/pull/2605/commits/0af48c5d5883f71b4880aa31bf5bbbf5f77a3ab9 because this was 1) unclear and 2) incorrect. I wanted a 5 minute buffer on either side.